docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory to `/app` can lead to issues with unwanted files being included in the container. Consider being more selective with the files you mount to avoid potential conflicts or security issues.